app/controllers/external_resources_controller.rb

```diff
@@ -59,8 +59,17 @@ def set_external_resource
 
     # Only allow a list of trusted parameters through.
     def external_resource_params
-      params.require(:external_resource).permit(:name, :website_url, :facebook_url,
-                                                :phone, :description, :display_on_website,
-                                                :youtube_identifier, :location_id, tags: [])
+      params.require(:external_resource).permit(
+          :description,
+          :display_on_website_end,
+          :display_on_website_start,
+          :facebook_url,
+          :location_id,
+          :name,
+          :phone,
+          :reviewed,
```

Comment:
Added the boolean  `reviewed` to help with workflows -- Admins can log in and see how many "unreviewed" resources there are...